src/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -62,6 +65,12 @@ export default function (on, options) {
             return
           }
 
+          if (options &&
```

Comment:
Can we pull this out to a util please, like we did with `optionsHaveIgnoredAtRule` in the [other PR](https://github.com/stylelint/stylelint/pull/1423/files)?